components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -393,6 +407,23 @@ - (UIColor *)messageTextColor {
   return self.header.messageTextColor;
 }
 
+- (void)setHeaderDividerColor:(UIColor *)headerDividerColor {
+  self.headerDividerView.backgroundColor = headerDividerColor;
+  [self.view setNeedsDisplay];
```

Comment:
Tip: Is this call necessary? I'm surprised that setting the `backgroundColor` isn't causing a layout pass. When I attempted to reproduce the need for `setNeedsDisplay`, the subview's updated `backgroundColor` is shown immediately.

At the least, please limit forced display updates to the headerDividerView's frame:

```objc
// Since the frame is a computed property, rely on the divider's bounds.
[self.view setNeedsDisplayInRect:[self.view convertRect:self.headerDividerView.bounds 
                                               fromView:self.headerDividerView]];
```

Reply:
This is no longer needed since we don't need `setNeedsDisplay`